governance/Community_spaces.md

```diff
@@ -0,0 +1,53 @@
+
+We encourage community members unsure about the suitability of their communications to contact a member of the Community Management Working Group, one of its chairs or to write to our inbox at uktrecommunity@gmail.com
+
+If you find this list too difficult to keep up with, we are setting up an announcements only mailing list.
+
+## Slack 
+We've moved! We are no longer maintaining the previous slack channel within the RSE workspace
+
+[UK TRE Community](https://uktrecommunity.slack.com) is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.
+Currently is where the Community Management Working Group coordinates, it is also used by chairs to discuss privately (on the rare ocassions that is needed, like sharing personal details).
+There are no specific channels, but these will be created as and if required by emerging interest or working groups.
+
+While everyone is welcome to make an announcement here it is NOT the expected place for them.
+
+## Github: Org, repositories and project tracker
+
+< general overview and reference to future specific doc on how to use the repo, or just do it here?>
+[uk-tre](https://github.com/uk-tre) is the GitHub organisation in which all the Community's content lives, and where work is organised and tracked for the CMWG and all working groups that want to do so.
+
+It is organised into different repositories, mainly:
+- website: hosts the content for the website
+- community-management: used by the CMWG for everything from governance to events planning and meeting notes
+- Working/Interest groups repositories: will be created as and if necessary, currently only 'packages' exists
+
+All work across repositories is tracked and openly discussed in the [TRE Community - Project board](https://github.com/orgs/uk-tre/projects/1), it allows to track the work done, in progress and proposals for future tasks.
+Working openly allows anyone to comment and contribute, you are welcome and encouraged to do so.
+
+Everyone is welcome to add issues to the corresponding repo with ideas and proposals, these will be added to the 'Ideas' column in 'Overview' and discussed when appropiatte for approval.
+
+The main views to be aware of are:
+- Overview: by far the most used one, it contains all issues by status.
+- By Deliverable: the fastest way to view all issues grouped by the expected outputs of the current funding call
+
+A more detailed guide on how to use the repositories and project will be added here: TBD
+
+## Announcements mailing list
+
+As we grow and the mailing list has more members it can be too much to keep up with all discussions, and discern between specific discussions and official announcements.
+Therefore we are creating a new announcement list, only list owners will be able to post messages and these will be restricted to official announcements (chiefly events).
+The average frequency of this list will not be highger than once a week.
```

Comment:
```suggestion
As the UK TRE community has grown (over 200 members on the existing mailing list!) the open nature of the list has led to some very valuable and insightful discussions.
However some people are finding it difficult to keep track of important announcements amongst the other discussions.
We've also found that some institutions are blocking the mailing list.

We therefore plan to creating a new _announcements only_ list where only the UK TRE co-chairs have permission to send messages.
These will be restricted to official announcements (chiefly events), and a summary of updates submitted by the community for inclusion.
We may also highlight particularly interesting discussions from the main list, and a more general roundup of relevant TRE news.
The average frequency of this list will not be higher than once a week.
We hope this provides a way for everyone to stay in touch regardless of how much time they're able to devote to the community.
```

Reply:
> We may also highlight particularly interesting discussions from the main list, and a more general roundup of relevant TRE news.

Do we want to be careful on how this may come across? (making it look like we decide what interesting is)

Reply:
And do we want to commit to a monthly roundup newsletter, or decide this later?

Reply:
I would leave it as 'may', we can say 'active' instead of 'interesting? As in we are highlighting discussions that got a lot of traction.
But I would not worry that much, the way I see it this is a proposal for discussion that we may adapt after the meeting